ui/src/app/edge/live/common/consumption/flat/flat.ts

```diff
@@ -28,20 +28,16 @@ export class FlatComponent extends AbstractFlatWidget {
     ]
 
     // Get consumptionMeterComponents
-    this.consumptionMeters = this.config.getComponentsImplementingNature("io.openems.edge.meter.api.SymmetricMeter")
+    this.consumptionMeters = this.config.getComponentsImplementingNature("io.openems.edge.meter.api.ElectricityMeter")
       .filter(component => component.isEnabled && this.config.isTypeConsumptionMetered(component));
 
     for (let component of this.consumptionMeters) {
       channelAddresses.push(
         new ChannelAddress(component.id, 'ActivePower'),
+        new ChannelAddress(component.id, 'ActivePowerL1'),
+        new ChannelAddress(component.id, 'ActivePowerL2'),
+        new ChannelAddress(component.id, 'ActivePowerL3'),
```

Comment:
i didnt read into the edge part, but the electricity meter is always writing phasechannels?